_includes/tutorials/anomaly-detection/ksql/code/docker-compose.yml

```diff
@@ -43,7 +43,7 @@ services:
       SCHEMA_REGISTRY_KAFKASTORE_BOOTSTRAP_SERVERS: 'broker:9092'
 
   ksqldb-server:
-    image: confluentinc/ksqldb-server:0.11.0
+    image: confluentinc/ksqldb-server:0.13.0
```

Comment:
```suggestion
    # In ksqlDB 0.14+, ksql-test-runner can experience a flaky timing issue with joins.
    # This test case is temporarily left at an earlier version.
    image: confluentinc/ksqldb-server:0.13.0
```

Reply:
@colinhicks stellar suggestion ❇️

Reply:
thanks for the review-within-review! Committed.